github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -25,11 +26,180 @@ presubmits:
         type: bare-metal-external
       containers:
         - image: quay.io/kubevirtci/bootstrap:v20210311-09ebaa2
+          env:
+            - name: TARGET
+              value: k8s-1.20
+            - name: KUBEVIRT_E2E_SKIP
+              value: Multus|SRIOV|GPU|Macvtap|Operator|sig-storage|sig-network|sig-compute
```

Comment:
Same here, `Multus` and `Macvtap` are not needed.

Reply:
Done